src/main/java/name/valery1707/kaitai/KaitaiMojo.java

```diff
@@ -222,15 +227,16 @@ private void executeInt() throws KaitaiException {
 		//Generate Java sources
 		Path output = mkdirs(this.output.toPath());
 		Path generatedRoot = KaitaiGenerator
-			.generator(kaitai, output, packageName)
-			.withSource(source)
-			.overwrite(overwrite)
-			.exactOutput(exactOutput)
-			.executionTimeout(executionTimeout)
-			.fromFileClass(fromFileClass)
-			.opaqueTypes(opaqueTypes)
-			.noVersionCheck(noVersionCheck)
-			.generate(logger);
+				.generator(kaitai, output, packageName)
+				.withSource(source)
+				.overwrite(overwrite)
+				.exactOutput(exactOutput)
+				.executionTimeout(executionTimeout)
+				.fromFileClass(fromFileClass)
+				.opaqueTypes(opaqueTypes)
+				.noVersionCheck(noVersionCheck)
+				.noAutoRead(noAutoRead)
+				.generate(logger);
```

Comment:
Please revert the indentation changes
```suggestion
			.generator(kaitai, output, packageName)
			.withSource(source)
			.overwrite(overwrite)
			.exactOutput(exactOutput)
			.executionTimeout(executionTimeout)
			.fromFileClass(fromFileClass)
			.opaqueTypes(opaqueTypes)
			.noVersionCheck(noVersionCheck)
			.noAutoRead(noAutoRead)
			.generate(logger);
```